db/migrate/20220711161804_remove_facebook_fields.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveFacebookFields < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :partners, :facebook_link
+    remove_column :users, :facebook_app_id
+    remove_column :users, :facebook_app_secret
```

Comment:
Would recommend adding a type here to allow it to roll back, just in case the migration on the server messes up and we do have to rollback for whatever reason :)